cmd/helm-sync/main.go

```diff
@@ -36,6 +36,14 @@ var (
 		"the name of the helm chart being synced")
 	flVersion = flag.String("version", os.Getenv(reconcilermanager.HelmChartVersion),
 		"the version of the helm chart being synced")
+	flValuesFiles = flag.String("values-files", os.Getenv(reconcilermanager.HelmValuesFiles),
+		"helm values files url")
+	flValuesValue = flag.String("values-v", os.Getenv(reconcilermanager.HelmValuesValue),
+		"the value of the helm chart values key value pair, will be used in helm template --set key=value")
+	flValuesKey = flag.String("values-k", os.Getenv(reconcilermanager.HelmValuesKey),
+		"the key of the helm chart values key value pair, will be used in helm template --set key=value")
```

Comment:
Seems weird to keep keys and values separated?

Reply:
Current solution is to process the values ( map[string]interface{} )in the util.go, combine all the keys together, all the values together and then pass to env. I am wondering if it would be better to pass the map to env directly.  Use Unstructured.MarshalJSON in util.go (unstructure -> byte[]) and then UnmarshalJSON (byte[] -> unstructure)in helm-sync container. Do you have any preference or suggestion?

Reply:
Separating the map k,v pair and then assembling them later is error prone. Could we combine the map as a flag with string type, such as `k1=v1,k2=v2`?

Reply:
Done